website/docs/reference/dbt-jinja-functions/local_md5.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "local_md5"
+id: "local_md5"
+---
+<Changelog>New in 1.4.0</Changelog>
+
+The `local_md5` context variable calculates an MD5 hash of the given string. It's called "local_md5" to emphasize that it runs locally in dbt (in jinja context) and not an MD5 SQL command.
+
+Usage:
+```
+{% set value_hash = local_md5("hello world") %}
+```
```

Comment:
Recommend including the "compiled" result as well. Two suggested options, one simpler and one fancier. Try both out, and get a feel for what feels good!

(Note the `<delete this line and replace with ```>` in the suggestions below, I couldn't figure out how to escape a codeblock within a codeblock)

1. Keeping it simpler:
```suggestion
```sql
-- source
{%- set value_hash = local_md5("hello world") -%}
'{{ value_hash }}'

-- compiled
'5eb63bbbe01eeed093cb22bb8f5acdc3'
<delete this line and replace with ```>
```

2. Getting a little fancier, using "tabs" (our custom React component):
```suggestion
<Tabs
  defaultValue="source"
  values={[
    { label: 'Source code', value: 'source', },
    { label: 'Compiled code', value: 'compiled', },
  ]
}>
<TabItem value="source">

<File name='my_model.sql'>

```sql
{%- set value_hash = local_md5("hello world") -%}
select '{{ value_hash }}' as my_md5_hash
<delete this line and replace with ```>

</File>

</TabItem>
<TabItem value="compiled">

<File name='my_model.sql'>

```sql
select '5eb63bbbe01eeed093cb22bb8f5acdc3' as my_md5_hash
<delete this line and replace with ```>

</File>

</TabItem>
</Tabs>
```